scripts/build-bin.sh

```diff
@@ -0,0 +1 @@
+CGO_ENABLED=0 go build -ldflags "-extldflags -static -s" -o /steve
```

Comment:
I wouldn't be able to run this directly without root. Could we create a `bin/` directory and do `-o ./bin/steve` instead? The docker container can copy it to `/` if it needs to.

Reply:
Should probably include the shebang at the top

Reply:
Good call, will adjust. I was a bit lazy in how I copied this from docker.

Reply:
Change made - thanks for the suggestion.